docs/config/dep-optimization-options.md

```diff
@@ -64,16 +64,17 @@ Set to `true` to force dependency pre-bundling, ignoring previously cached optim
 
 ## optimizeDeps.disabled
 
+- **Deprecated**
 - **Experimental:** [Give Feedback](https://github.com/vitejs/vite/discussions/13839)
 - **Type:** `boolean | 'build' | 'dev'`
 - **Default:** `'build'`
 
-Disables dependencies optimizations, `true` disables the optimizer during build and dev. Pass `'build'` or `'dev'` to only disable the optimizer in one of the modes. Dependency optimization is enabled by default in dev only.
+This option is deprecated. As of Vite 5.1, pre-bundling of dependencies during build have been removed. Setting `optimizeDeps.disabled` to `true` or `'dev'` disables the optimizer, and configured to `false` or `'build'` leaves the optimizer during dev enabled.
 
-:::warning
-Optimizing dependencies in build mode is **experimental**. If enabled, it removes one of the most significant differences between dev and prod. [`@rollup/plugin-commonjs`](https://github.com/rollup/plugins/tree/master/packages/commonjs) is no longer needed in this case since esbuild converts CJS-only dependencies to ESM.
+Use `optimizeDeps.noDiscovery` instead to disallow automatic discovery of dependencies and leave `optimizeDeps.include` undefined or empty to disable completely the optimizer instead.
```

Comment:
```suggestion
To disable the optimizer completely, use `optimizeDeps.noDiscovery: true` to disallow automatic discovery of dependencies and leave `optimizeDeps.include` undefined or empty.
```

small nit: Removed the double `instead` and rephrased it a bit so it's easier to scan for a migration guide